### DISCUSSION TITLE
Dynamic image source

### DISCUSSION THREAD

**Discussion by dmzone** [alpinejs/alpine#2136] [1. Help] [answered]
Hi all, is it possible to "inject" a string in an image path source?

```javascript
<ul x-data="{langs:['en','fr','de']}">
    <template x-for="(LANG_STRING, index) in langs" :key="index">
        <li>
            <img x-bind:src="./images/flag_LANG_STRING.png">
        </li>
    </template>
</ul>
```

**Comment by Tim-Wils** [comment, marked as answer]
Yes, this is possible :) https://jsfiddle.net/71vosxa9/
The binded src uses a javascript expression, so you will have to use a js-string expression with a variable to make it work.
```<img x-bind:src="`./images/flag_${LANG_STRING}.png`">```

**Reply by dmzone** [reply]
@Tim-Wils 
that was my first try... how is possible that template literals won't work?

**Reply by Tim-Wils** [reply]
@dmzone Did you see my JSFiddle? It works.
Sreenshot of the source:
![afbeelding](https://user-images.githubusercontent.com/90032/135070106-96e00e49-1d3c-4e5f-a9f1-10e7e1b18e9c.png)
